### PR TITLE
Build manifest for org.wesnoth.Wesnoth

### PR DESCRIPTION
**Pull request by cosimoc**
Based on the Freedesktop 1.6 SDK.

### CODE REVIEW

File: org.wesnoth.Wesnoth.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ],
+    "modules": [
+        "shared-modules/SDL/SDL-1.2.15.json",
+        "shared-modules/smpeg/smpeg-0.4.5.json",
+        "shared-modules/SDL/SDL_mixer-1.2.12.json",
+        "shared-modules/SDL/SDL_image-1.2.12.json",
+        "shared-modules/SDL/SDL_ttf-2.0.11.json",
+        "shared-modules/SDL/SDL_net-1.2.8.json",
+        {
+            "name": "boost",
```
Comment (Barthalion) [inline org.wesnoth.Wesnoth.json]: As usually, `simple` seems better suited.

**Comment by Barthalion** [comment]
As my comment is pretty much irrelevant, started a test build: https://flathub.org/builds/#/builders/5/builds/274

**Comment by cosimoc** [comment]
@Barthalion Looks like that worked... I also pushed a commit to change the manifest to use the simple buildsystem like you suggested.

**Comment by Barthalion** [comment]
Merged, thanks!

**Pull request closed without merging**